app/models/course.rb

```diff
@@ -37,7 +37,7 @@ class Course < ApplicationRecord
   after_initialize :set_defaults
 
   before_discard do
-    raise "You cannot delete a running course (Course: #{self}, Provider: #{provider_id})" if ucas_status != :new
+    raise "You cannot delete a running course (Course: #{self}, Provider: #{provider_id})" unless ucas_status == :new
```

Comment:
dont `self` uses the `to_s ` that gives provider stuff as well?

Reply:
Yeah self will use the to_s

Reply:
can you make the message nice please...